<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ProjectLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="newProjectButton">
       <property name="text">
        <string>New Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveProjectButton">
       <property name="text">
        <string>Save Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadProjectButton">
       <property name="text">
        <string>Load Project</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="CurrentProjectLayout" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>29</y>
      <width>1311</width>
      <height>781</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="CurrentProjectButtonsLayout">
      <item>
       <widget class="QPushButton" name="moduleSettingsButton">
        <property name="text">
         <string>Module settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moduleFilesetButton">
        <property name="text">
         <string>Module fileset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moduleParametersButton">
        <property name="text">
         <string>Module parameters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Module connections</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="windowForWidgets" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>1281</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
